package.json

```diff
@@ -19,6 +19,8 @@
     "class-validator": "^0.13.1",
     "cli-table3": "0.6.0",
     "cli-ux": "5.4.9",
+    "colors": "^1.4.0",
```

Comment:
We already use [chalk](https://www.npmjs.com/package/chalk) for coloring logs. Search `chalk.green`. Surprised to see its not in our package.json dependencies. We should probably add.